uktrade/great-cms
# Gls 113 country factsheets

Adds and populates `iso2` codes to the countries database table.

Adds an extra link to CTA links on Country Guide pages. The last 2 CTA links are generated based on the country ISO2:
  - View live export opportunities
  - Find latest export events
  - View latest trade statistics
  - Find an online marketplace
  - Check duties and customs
  - Check for trade barriers

Links above are always shown regardless of device screen size (used to be hidden on mobile). A new header, `More on {country}` shows above the links.

Also adds a management command to update all existing links and add new ones (as well as generate the Trade barriers link), and reorder, as above. This can be run once the migration for the extra CTAs has run:
```
make manage organise_country_guide_ctas
```

A `--dry-run` option is available for this command, which will provide a summary of changes without updating the actual data. This may help highlighting incorrect data (some market guides incorrectly use the `Find export events` link text for export opportunities, e.g. https://www.great.gov.uk/export-opportunities/opportunities?s=&areas%5B%5D=chile&commit=Find+opportunities).

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GLS-113

### Reviewing help


### Housekeeping

- [ ] Ran the `make manage download_geolocation_data` command

### Merging
